book/src/animation/definition.md

```diff
@@ -0,0 +1,13 @@
+# Definition
+
+Animations can be defined for objects made of a single entity, or complex objects made up of multiple entities.
+
+Right now we do not have a tutorial for defining an animation from scratch, but take a look at the following resources to get an idea of how to write one:
```

Comment:
We don't have a tutorial for animations. Do you mind opening an issue for that, and reference this line so we don't forget to change it when we do?

Reply:
Certainly :slightly_smiling_face: #637